data/languagefiles/en.yaml

```diff
@@ -89,21 +89,26 @@ second:
 
 ago:
     - ago
+in:
+    - in
+    - from now
 
 simplifications:
```

Comment:
Maybe we should also add simplification for time units such as fortnight, decade and century?